app/controllers/api/base_controller.rb

```diff
@@ -2,6 +2,10 @@ module Api
   class BaseController < ActionController::Base
     protect_from_forgery with: :null_session
 
+    rescue_from(ActionController::ParameterMissing) do |err|
+      render json: {missing_param: err.param}, status: :bad_request
```

Comment:
👍  for the *symbolic* status.

`{ ... }` content should be padded by spaces. I think rubocop would complain about it, try to run it if you have time, not a blocker though.